packages/client/modules/userDashboard/components/OrgTeams/OrgTeams.tsx

```diff
@@ -50,8 +49,8 @@ const OrgTeams = (props: Props) => {
   const [sortBy, setSortBy] = useState<SortField>('lastMetAt')
   const [sortDirection, setSortDirection] = useState<SortDirection>('desc')
 
-  const {allTeams, tier, viewerTeams, allTeamsCount, hasPublicTeamsFlag} = organization
-  const showAllTeams = allTeams.length === allTeamsCount || hasPublicTeamsFlag
+  const {allTeams, tier, viewerTeams, allTeamsCount} = organization
+  const showAllTeams = allTeams.length === allTeamsCount
```

Comment:
+1 `showingAllTeams` or `allTeamsShown` would be easier to understand

Reply:
Updated 👍